stdlib/string.gr

```diff
@@ -38,6 +38,8 @@ provide enum Encoding {
 
 exception MalformedUnicode
 
+exception RangeError
```

Comment:
I don't think we should have a new exception. This should just throw `InvalidArgument`. I'm also fine with the empty string behavior, but I think throwing is a little better.

Reply:
I had already implemented it using `InvalidArgument` I had just forgot to remove the `RangeError` exception I added before making that decision. 

Made that fix now though.